sdk/include/ndk/ldrtypes.h

```diff
@@ -196,26 +200,38 @@ typedef struct _LDR_ENUM_RESOURCE_INFO
 //
 // DLL Notifications
 //
+#define LDR_DLL_NOTIFICATION_REASON_LOADED 1
```

Comment:
```suggestion
#define LDR_DLL_NOTIFICATION_REASON_LOADED   1
```
(indenting the value with the next one, as it is customary in the NDK)